packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -89,6 +89,18 @@
 					</template>
 				</tera-operator>
 			</tera-canvas-item>
+			<tera-operator-menu
+				v-if="nodeWithOpenedMenu && isOutputMenuShown"
+				:node="nodeWithOpenedMenu"
+				:style="{
+					top: `${currentPortPosition.y - 18}px`,
+					left: `${currentPortPosition.x + 40}px`
+				}"
+				@mouseenter="() => (isMenufocused = true)"
+				@mouseleave="() => (isMenufocused = false)"
+				@focus="() => (isMenufocused = true)"
+				@blur="() => (isMenufocused = false)"
+			/>
```

Comment:
This should not be a first-class item on the canvas, it makes things more complicated to manage the more floating-things we have.

Reply:
Yeah, should be in `tera-operator` I assumed I was reading that file actually